ui/src/app/pages/analysis/leonardo-app-launcher.tsx

```diff
@@ -51,8 +51,8 @@ import {
 } from './util';
 
 export enum LeoApplicationType {
```

Comment:
I don't like that we have 2 concepts of "application type" but we can distinguish them better at least